pkgs/firehose/lib/src/health/health.dart

```diff
@@ -129,9 +129,7 @@ class Health {
       };
 
   Future<HealthCheckResult> validateCheck() async {
-    //TODO: Add Flutter support for PR health checks
```

Comment:
Most of the uses of `use-flutter:` were for health checks - and the health check firehose action ignores that.